src/roman_datamodels/random_utils.py

```diff
@@ -138,6 +138,69 @@ def generate_array_float32(size=(4096, 4096), min=None, max=None, units=None, se
     return array
 
 
+def generate_array_complex128(size=(32, 286721), min=None, max=None, shape=None, seed=None):
+    """
+    Uniformly at random generate a complex array of the specified size.
+
+    Parameters
+    ----------
+    size: tuple
+        The size of the array to generate.
+
+    min: tuple (or None)
+        Min value for the sample axis.
+            - If None and shape is "square", then (-1.0, -1.0).
+            - If None and shape is "disk", then (0.0, 0.0).
+
+    max: tuple (or None)
+        Max value for the sample axis.
+            - If None and shape is "square", then (1.0, 1.0).
+            - If None and shape is "disk", then (1.0, 2pi).
```

Comment:
This says 0 to 2pi, but the code goes from -pi to pi.

Reply:
This does not matter. It's randomly selecting the arg of the complex number. You will get identical results.